intro-Cloud/ch_ConfigureInstances.tex

```diff
@@ -1,41 +1,73 @@
 \chapter{Configure access and security for instances}\label{cha:conf-access-secur}
+The security and accessibility of your cloud resources is governed by
+a few different aspects, which we discuss more detail in the following
+sections:
+\begin{itemize}
+\item Each cloud project can use one floating IP, a public IP address
+  which you'll need to connect to the resources you want to access.
+\item By default, the UGent firewall blocks most IP addresses and
+  ports.  Only the port range 50000-60000 for the floating IP
+  addresses is open by default.  Contact \cloudinfo if you need to
+  access other ports from the outside world.
+\item Instances must connect to the project's \_vm network in order to
+  communicate.
+\item The OpenStack environment has its own internal firewalls, which
+  block most ports of your instances.  If you want to access specific
+  ports of your instances, you must create ``Security Groups'' which
+  allow access to those ports.
+\item If you want to access your instances using SSH key
```

Comment:
Just change this item a bit. Users do not have to upload their keys because this is sync with the account portal, so they only have to upload their keys to the account portal as they do for HPC (the sync script will update the user's openstack keys automatically from there). Maybe just include the link from the hpc doc to know how to upload keys to the vsc system (section 2.2 I think)

Reply:
This section does not talk about uploading new keys.  You still need to ask OpenStack to inject your key to instances you create, regardless of how that key was imported into OpenStack (by the user themselves or by the synchronization with the account portal).  I've added a sentence mentioning the account keys to make it more clear, hopefully (all of this discussed in more detail in other sections, this is just a general introduction to the chapter).

Reply:
ok, in that case I would include a reference to the specific ssh section here, just to avoid any confusion (something like `for more info about ssh keys injection see \ref{sec:ssh-keypairs}`). In that case they can go directly to the detailed info.

Reply:
Ok, I've shortened the introduction (no need to duplicate information here) and added references to the relevant sections.  I've also reordered the sections to match the order of in which they are introduced, instead of the arbitrary order in which they appeared until now.